portfolio/src/main/webapp/script.js

```diff
@@ -13,16 +13,21 @@
 // limitations under the License.
 
 /**
- * Adds a random greeting to the page.
+ * Changes picture and caption of img.
  */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+function changePicture() {
+  const pictureContainer = document.getElementById('pet-container');
+  const captionContainer = document.getElementById('pet-caption');
+  const petPictures =
+      ['/images/IMG_7208.JPG', '/images/IMG_7209.JPG'];
 
-  // Pick a random greeting.
-  const greeting = greetings[Math.floor(Math.random() * greetings.length)];
-
-  // Add it to the page.
-  const greetingContainer = document.getElementById('greeting-container');
-  greetingContainer.innerText = greeting;
-}
+  // Make sure you don't get the same picture twice in a row.
+  if((pictureContainer.src).charAt(65)==9){
```

Comment:
Could you make constants with meaningful variable names for these numbers? It will make it easier to understand what's happening here.